src/js/UserManager.js

```diff
@@ -0,0 +1,18 @@
+import { createUserManager } from 'redux-oidc'
+
+const userManager = createUserManager({
+  authority: 'https://192.168.99.100:8443/auth/realms/hmda',
```

Comment:
Cool by me.  Added it to https://github.com/cfpb/hmda-platform/issues/661#issuecomment-257983236.